io-engine/src/subsys/nvmf/transport.rs

```diff
@@ -32,16 +34,19 @@ use crate::{
 static TCP_TRANSPORT: Lazy<CString> =
     Lazy::new(|| CString::new("TCP").unwrap());
 
-pub async fn add_tcp_transport() -> Result<(), Error> {
+pub static RDMA_TRANSPORT: Lazy<CString> =
+    Lazy::new(|| CString::new("RDMA").unwrap());
```

Comment:
btw can't these be lowercase? then the display would just work?

Reply:
It'll may be work for the case where we use this const to just generate `spdk_nvme_transport_id` ourselves and hand marshal the `trstring` there. But this const is also used in transport create path where spdk does some validations based on this, hence don't want to touch it.